uproot/hist.py

```diff
@@ -319,3 +678,33 @@ def hist(numbins, low, high, name=None, title=None, values=None, allvalues=None,
         out.fillall(filldata)
 
     return out
+
+def hist2d(xnumbins, xlow, xhigh, ynumbins, ylow, yhigh, name=None, title=None, values=None, allvalues=None, xfilldata=None, yfilldata=None):
+    out = TH1()
```

Comment:
apologies for the drive-by comment, but:

`TH1()` ?
shouldn't this be `TH2()` ?

Reply:
Nice catch, thanks!